website/assets/apps/megamap/components/SearchView.jsx

```diff
@@ -452,16 +464,26 @@ export default class SearchView extends Component {
                     />
                   </div>
                 </Panel>
-            </div>
-            <header className="searchView__header">
-              <button
-                type="submit"
-                title="Search"
-                disabled={!this.state.searchEnabled}
-              >UPDATE RESULTS
-              </button>
-              <span></span>
-            </header>
+              </div>
+              <header
+                className={classNames(
+                  'searchView__header',
+                  'searchView__header--interactive',
+                  { 'searchView__header--disabled': !this.state.searchEnabled },
+                )}
+              >
+                <button
+                  type="submit"
+                  title="Search"
+                  className="searchView__update-results"
+                  disabled={!this.state.searchEnabled}
+                >
+                  <span>
+                    Update Results
```

Comment:
The span here should be ``#ffffff`` when Active or On Hover. Right now it's ``#eee8e4``